src/handler/post.ts

```diff
@@ -0,0 +1,34 @@
+import 'dotenv/config';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { postTechRecord } from '../services/database';
+import logger from '../util/logger';
+
+export const handler = async (
+  event: APIGatewayProxyEvent,
+): Promise<{ body: unknown; statusCode: number }> => {
+  logger.info('Post end point called');
+  try {
+    if (event.body === null) {
+      return {
+        statusCode: 400,
+        body: JSON.stringify({ error: 'Body is not a valid TechRecord' }),
+      };
+    }
+    // TODO to use proper type when we have them
```

Comment:
So, this is where we would like to leverage the npm package we created to runtime-validate the input. IF everything was setup correctly in the npm package, it should be somewhat trivial to validate the body of the request.